internal/host/validator.go

```diff
@@ -1102,15 +1102,15 @@ func checkDomainNameResolution(c *validationContext, domainName string) Validati
 	return ValidationFailure
 }
 
-func printIsDomainNameResolvedCorrectly(c *validationContext, status ValidationStatus, domainName string) string {
+func printIsDomainNameResolvedCorrectly(c *validationContext, status ValidationStatus, domainName string, errorMsg string) string {
```

Comment:
The name `errorMsg` is a bit confusing, consider renaming to `destination` or `loadBalancer`

Reply:
Agree, changed to `destination`